charts/xp-management/Chart.yaml

```diff
@@ -18,4 +18,4 @@ maintainers:
 - email: caraml-dev@caraml.dev
   name: caraml-dev
 name: xp-management
-version: 0.2.4
+version: 0.3.0
```

Comment:
why do we increase this version if do not add any rules to this chart templates list?